dynamic-demo-plugin/OWNERS

```diff
@@ -0,0 +1,23 @@
+reviewers:
+  - dtaylor113
+  - jcaianirh
+  - jeff-phillips-18
+  - jhadvig
+  - kyoto
+  - rhamilto
+  - spadgett
+  - TheRealJon
+  - zherman0
+approvers:
+  - andrewballantyne
+  - bparees
```

Comment:
I'd remove @bparees here

Reply:
@bparees I assume you don't want to be pinged for approvals :)

Reply:
OK, removed

Reply:
fine by me :)